releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,37 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 5.3.0-preview - July 27, 2022
+* Added support for fail-over mechanism leveraging geo-replication support for Azure App Configuration stores. To use this capability a new API `Connect(List<Uri> endpoints, TokenCredential credential)` has been added. A sample application using fail-over mechanism may look like the following -
```

Comment:
We should also document the behaviour when multiple endpoints are registered. A user might be interested in knowing things like 
- which errors are considered failoverable
- what is the max delay that they can expect if all endpoints fail
- what is the order in which endpoints will be tried

All this may be too complicated for release notes, so maybe we can open an issue in our provider repo explaining the goal of this API, and link that issue to the release notes?

Reply:
What we added is a new API. Fail-over is what it does. I rephrased the sentence a little bit and (somewhat) incorporated Avani's feedback. Suggest below:

- Added the following new API for the App Configuration geo-replication support.

  ```cs
  public AzureAppConfigurationOptions Connect(IEnumerable<Uri> endpoints, TokenCredential credential)
  ```

  The new API allows you to provide an ordered list of replica endpoints of your App Configuration store. The App Configuration provider will fail over to other replicas when a replica is not accessible (either the service is unavailable or there are networking issues). A sample code snippet may look like the following.

Reply:
I think we can publish an official doc showing how to leverage geo replication with provider libraries. These release notes are good enough if we plan to release an official doc in future.

Reply:
I like @zhenlan's suggestion. Updated.
We can link the official doc once it is available, but I think the link to the example would help people get started.